arbitrum-docs/launch-orbit-chain/orbit-sdk/introduction.md

```diff
@@ -0,0 +1,63 @@
+
+### Capabilities of the SDK:
+- **Configuration and Deployment**: Simplify the process of configuring and deploying the core contracts needed for an Orbit chain.
+- **Initialization and Management**: After deployment, utilize the SDK to initialize your chain and adjust configuration parameters as necessary.
+- **Ongoing Support and Updates**: Look forward to future releases that will bring new features, such as tools for creating custom dashboards and monitoring systems for your Orbit chain.
+
+By choosing the Arbitrum Orbit SDK, you're not just accessing a set of development tools; you're unlocking a universe of opportunities to create bespoke blockchain projects tailored to your needs. Stay tuned for more exciting updates and features as we continue to enhance the Arbitrum Orbit SDK, making it an even more powerful resource for developers.
+
+Below, you will find detailed information about the steps for setting up an Orbit chain and how to utilize the Orbit SDK throughout the process:
+
+### 1. Orbit Chain Deployment
+
+A primary function of the Orbit SDK is the deployment of a new Orbit chain. The initial step in setting up an Orbit chain involves determining the type that best fits your needs. We offer four distinct types of Orbit chains:
+
+1. **Rollup Orbit Chains**: These are the most straightforward Orbit chains. In this type, transaction data is batched, compressed, and posted to the parent chain, akin to the Arbitrum One chain as a rollup chain. Detailed steps for deploying a rollup chain can be found [here](deployment-rollup.md).
+
+2. **Anytrust Orbit Chains**: Anytrust Orbit chains share a similar logic with rollup chains but with a key difference: batches are not posted directly to the parent chain. Instead, they are stored by a Data Availability Committee (DAC) to reduce transaction fees. The main differences between an Anytrust chain and a rollup chain are:
+   
+   - **Data Availability**: Anytrust chains utilize a unique approach, with members of a Data Availability Committee responsible for keeping transaction data, whereas rollup chains send the data back to the parent chain.
+
+   - **Security Guarantees**: Anytrust chains offer different security levels that might be more suited for specific applications, like gaming and social media.
+
+   - **Fee Cost**: Anytrust chains are significantly cheaper than Rollup chains because there's no need to send data to the parent chain, facilitating cheaper transaction fees for applications.
```

Comment:
```suggestion
2. **Anytrust Orbit Chains**: Anytrust Orbit chains are implementations of the Rollup protocol that lower costs by accepting a mild trust assumption. <a data-quicklook-from="arbitrum-anytrust-protocol">AnyTrust</a> chains rely on an external Data Availability Committee (DAC) to store data and provide it on-demand instead of using its{' '} <a data-quicklook-from="parent-chain">parent chain</a> as the Data Availability (DA) layer. Instead, they are stored by a Data Availability Committee (DAC) to reduce transaction fees.
```

Reply:
Merged.